src/arcaflow_plugin_sdk/schema.py

```diff
@@ -2577,8 +2577,112 @@ def _to_openapi_fragment(
         return {"$ref": "#/components/schemas/" + self.id}
 
 
+StrInt = typing.Union[int, str]
+
+
 @dataclass
-class OneOfStringSchema(_JSONSchemaGenerator, _OpenAPIGenerator):
+class OneOfSchema(_JSONSchemaGenerator, _OpenAPIGenerator):
+    types: Dict[
+        StrInt, typing.Annotated[_OBJECT_LIKE, discriminator("type_id")]
+    ]
```

Comment:
Should (could?) this instead be
```python
    types: typing.Union[
        Dict[
            str, typing.Annotated[_OBJECT_LIKE, discriminator("type_id")]
        ],
        Dict[
            int, typing.Annotated[_OBJECT_LIKE, discriminator("type_id")]
        ]
    ]
```
That is, as far as I can make out, `types` is _either_ a `dict` with `str` keys _or_ a `dict` with `int` keys -- it's never a `dict` with _both_.

Reply:
yeah good point

Reply:
Glad it worked!